tools/console.js

```diff
@@ -28,6 +28,9 @@ var wordwrap = require('wordwrap');
 
 var PROGRESS_DEBUG = !!process.env.METEOR_PROGRESS_DEBUG;
 var FORCE_PRETTY=undefined;
+var CARRIAGE_RETURN =
```

Comment:
Maybe something like Console.clearLine()? When I think of carriage return I think of a new line but I could be wrong